evc-sumo/docker/build-image.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+#  Copyright (C) 2023 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+USERNAME=usdotfhwastol
+
+
+cd "$(dirname "$0")"
+# IMAGE=$(./get-image-name.sh | tr '[:upper:]' '[:lower:]')
+IMAGE=econolite-virtual-controller
+
+echo ""
+echo "##### $IMAGE Docker Image Build Script #####"
+echo ""
+
+while [[ $# -gt 0 ]]; do
```

Comment:
If you define the environment variable like the comment above, this should be unecessary. The defined environment variable based on the docker ARG will just propagate into this file and be available as a regular environment variable.